processes/procstat.go

```diff
@@ -76,18 +76,18 @@ func (psc *procStatsCollector) GetStats(procPath string) (map[string][]Proc, err
 			fstat := filepath.Join(procPath, file.Name(), procStat)
 			procStat, err := ioutil.ReadFile(fstat)
 			if err != nil {
-				return nil, err
+				continue
```

Comment:
Hello @obourdon. Besides the lack of logging in these place that @katarzyna-z mentioned, it looks good to me. I prepared a pull request (https://github.com/obourdon/snap-plugin-collector-processes/pull/1/files) to your repo with logs in these places. Logs are available when snapd is running with log level set to 1.